req-tools.txt

```diff
@@ -1,4 +1 @@
-github.com/fzipp/gocyclo/cmd/gocyclo@v0.6.0
-golang.org/x/tools/cmd/goimports@v0.1.11
```

Comment:
If goimport target will be returned, could you, please return it back too?

Reply:
Depends on your answer on https://github.com/emacsway/qualifying-grade/pull/6#discussion_r932730161

Reply:
Ok, let's just add --fix option to golangci-lint run cmd.